generators/client/templates/react/src/test/javascript/spec/entry.ts.ejs

```diff
@@ -1,36 +0,0 @@
-<%#
- Copyright 2013-2018 the original author or authors from the JHipster project.
-
- This file is part of the JHipster project, see https://www.jhipster.tech/
- for more information.
-
- Licensed under the Apache License, Version 2.0 (the "License");
- you may not use this file except in compliance with the License.
- You may obtain a copy of the License at
-
-      http://www.apache.org/licenses/LICENSE-2.0
-
- Unless required by applicable law or agreed to in writing, software
- distributed under the License is distributed on an "AS IS" BASIS,
- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- See the License for the specific language governing permissions and
- limitations under the License.
--%>
-import 'core-js';
-import * as sinon from 'sinon';
-import * as chai from 'chai';
-import * as chaiEnzyme from 'chai-enzyme';
-import { configure } from 'enzyme';
```

Comment:
How is enzyme configured now?

Reply:
doesn't seem needed https://github.com/airbnb/enzyme/blob/master/docs/guides/jest.md

Reply:
great